### DISCUSSION TITLE
How to disable the certificate verification in Weblate Docker based installation?

### DISCUSSION THREAD

**Discussion by phandja** [WeblateOrg/weblate#7135] [General]
Is it possible to add the correct way to disable the certificate verification on Docker based installation as we use to do in setting.py?
---
import ldap
from django_auth_ldap.config import LDAPSearch, GroupOfNamesType, NestedActiveDirectoryGroupType
ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)
---

**Comment by nijel** [comment]
Use custom configuration for that, see https://docs.weblate.org/en/latest/admin/install/docker.html#further-configuration-customization

**Reply by phandja** [reply]
It worked! Thank you @nijel!
